hera_sim/visibilities/prisim_wrapper.py

```diff
@@ -0,0 +1,334 @@
+from __future__ import print_function
+import os
+import warnings
+
+import numpy as np
+import pyuvdata
+from astropy import units as u
+from astropy.coordinates import SkyCoord, FK5, AltAz, EarthLocation
+from astropy.io import fits
+from astropy.time import Time
+from cached_property import cached_property
+
+try:
+    from astroutils import catalog, geometry, mathops
+    from prisim import interferometry as intf
+    import prisim
+except ImportError:
+    raise ImportError("To use the PRISim wrapper, you will need to install prisim!")
+
+from .simulators import VisibilitySimulator
+
+PRISIM_PATH = prisim.__path__[0]
+
+
+class PRISim(VisibilitySimulator):
+    diffuse_ability = False
+
+    def __init__(self, aux_file_loc='prisim_roi_file', fov_radius=90.0,
+                 beam_pol='x', beam_interp='cubic', precision='single', **kwargs):
+        self.fov_radius = fov_radius
```

Comment:
Let's raise a `NotImplementedError` in here for now

Reply:
Raised the error at the beginning of __init__.